pydev_ipython/qt_loaders.py

```diff
@@ -109,23 +118,24 @@ def has_binding(api):
     module_name = {
         QT_API_PYSIDE: "PySide",
         QT_API_PYSIDE2: "PySide2",
+        QT_API_PYSIDE6: "PySide6",
         QT_API_PYQT: "PyQt4",
         QT_API_PYQTv1: "PyQt4",
         QT_API_PYQT_DEFAULT: "PyQt4",
         QT_API_PYQT5: "PyQt5",
+        QT_API_PYQT6: "PyQt6",
     }
     module_name = module_name[api]
 
-    import importlib
+    import imp
```

Comment:
This seems backward... `importlib` should be preferred over `imp` which was deprecated (so, I think the version using `importlib` should be kept -- unless you see a real reason for the switch).

Reply:
I might have merged this wrong. I'll switch it back.